Makefile

```diff
@@ -23,8 +28,7 @@ server/api/mocks/starship_action.go:
 	@retool do mockery -name Starfriends_ListStarshipActionsServer -dir server/proto  -output ./server/api/mocks
 
 server/proto/%.pb.go: proto/%.proto
-	$(info Generating go and gRPC protos...)
-	@docker run -v $(PWD):/defs namely/protoc-all:1.9 -d proto -l go -o server/proto
+	$(info DEPRECATED: use "make generate-pb" instead)
```

Comment:
I ditched this because you don't really need to care about the specific file since we just generate the entire directory. Much simpler to do `make generate-pb`

Reply:
This was here so that running tests built these as needed. Do we now have to run two steps?

Reply:
Well we can make the test target just invoke the generate-pb one too right?

Reply:
You can, but then it'll always run instead of only when needed. It's no biggie if you really prefer the generate-pb thing